### PR TITLE
completed HW 3.1 (Maxim Litvinenko)

### PR DESCRIPTION
**Pull request by Massonus**

### CODE REVIEW

File: Massonus/HW3/HW3.1.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+48.Unless explicitly silenced.
+49.In the face of ambiguity, refuse the temptation to guess.
+50.There should be one-- and preferably only one --obvious way to do it.
+51.Although that way may not be obvious at first unless you're Dutch.
+52.Now is better than never.
+53.Although never is often better than *right* now.
+54.If the implementation is hard to explain, it's a bad idea.
+55.If the implementation is easy to explain, it may be a good idea.
+56.Namespaces are one honking great idea -- let's do more of those!"""
+
```
Comment (kolyasalubov) [inline Massonus/HW3/HW3.1.py:57]: good job

File: Massonus/HW3/HW3.2.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+number = input("Input a four-digit number: ")
+numbers = list(map(int, number))
+result = numbers[0] * numbers[1] * numbers[2] * numbers[3]
+print("Product of your number:", result)
+
+reverse = int(number[::-1])
+print("Reverse:", reverse)
+
+numbers = sorted(numbers, reverse=False)
+sorted_numbers = int(str(numbers[0]) + str(numbers[1]) + str(numbers[2]) + str(numbers[3]))
```
Comment (kolyasalubov) [inline Massonus/HW3/HW3.2.py:10]: excellent idea

File: Massonus/HW3/HW3.3.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+a = 23
+b = 45
+print("a =", a, "b=", b)
+
```
Comment (kolyasalubov) [inline Massonus/HW3/HW3.3.py:4]: good idea
look at this implementation
a, b = b, a

**Review by kolyasalubov — APPROVED** [review]

File: Massonus/HW3/HW3.1.py (hunk 1 above)
Comment (Massonus) [reply Massonus/HW3/HW3.1.py:57]: great!

**Pull request merged**